src/integrations/bluetooth-low-energy/bluetooth-low-energy.service.ts

```diff
@@ -83,7 +83,7 @@ export class BluetoothLowEnergyService extends KalmanFilterable(Object, 0.8, 15)
       this.seenIds.add(tag.id);
     }
 
-    if (this.isOnWhitelist(tag.id)) {
+    if (this.isOnWhitelist(tag.id) || this.isOnWhitelist(tag.name)) {
```

Comment:
I think my current ideal wish for how the configuration could look would be similar to what Home Assistant does... you could either have a list of strings the represent IDs (backward compatibility) or a list of objects with `type` and `value`. For the type you would then put `name` if the whitelist item should be applied to the tag name. E.g.:

```yaml
bluetoothLowEnergy:
  whitelist:
    - abcd-1-2
    - type: name
      value: my phone
```